src/main.rs

```diff
@@ -30,6 +30,13 @@ fn main() {
         .long_help("Bash and Zsh supports returning codes for each process in a pipeline.")
         .multiple(true);
 
+    let terminal_width_arg = Arg::with_name("terminal_width")
+        .short("t")
```

Comment:
I would prefer `-w` for the argument
```suggestion
        .short("w")
```